app/controllers/home_controller.rb

```diff
@@ -100,7 +100,11 @@ def studentInfo
       return nil
     end
 
-    atm = Qroster.where(quiz_id:qID,correct_resp:true).count(:attempts)
```

Comment:
The original line (this) should be correct because I'm not truly trying to count "attempts", just the number of completed rows in Qroster for the prof.

Because later it's used for weighting the average, which is done by (# of completed rows for this studentID / total # of completed rows for this prof)

Reply:
Sounds good, I'll revert it back

Reply:
Sounds good, I'll revert it back